databroker_pack/_pack.py

```diff
@@ -122,8 +122,7 @@ def export_uids(
                 )
                 for root, set_ in files.items():
                     accumulated_files[root].update(set_)
-                    for filename in set_:
-                        file_uids[uid].append((root, filename))
+                    file_uids[uid] = [(root, filename) for filename in set_]
```

Comment:
As written, each iteration through the for loop in Line 123 will overwrite the list previous iterations. I think the list has to be defined above Line 123 and the `extend`-ed here.